### PR TITLE
fix: tuples type generation support (#1011)

### PR DESCRIPTION
**Pull request by liangskyli**
## Changes
fix bug: #1011

## How to Review

Tests

## Checklist

- [x] Unit tests updated
- [ ] README updated
- [ ] `examples/` directory updated (if applicable)

### CODE REVIEW

File: test/schema-object.test.ts [hunk 1]
```diff
@@ -109,6 +109,12 @@ describe("Schema Object", () => {
         expect(generated).toBe("(string)[]");
       });
 
+      test("tuple array", () => {
+        const schema: SchemaObject = { type: "array", items: [{ type: "string" },{"type": "number"}],minItems:2,maxItems:2 };
```
Comment (drwpow) [inline test/schema-object.test.ts:113]:
```suggestion
        const schema: SchemaObject = { type: "array", prefixItems: [{ type: "string" },{"type": "number"}],minItems:2,maxItems:2 };
```

**Review by drwpow — CHANGES_REQUESTED** [review]
Thank you very much for the PR! I’m in favor of merging this, but would like this to require usage of `prefixItems` [according to the spec](https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation) rather than `items`

**Comment by liangskyli** [comment]
@drwpow : You can add support for prefixItems, but I think items also need support！
see: https://json-schema.org/draft/2020-12/release-notes.html
image
because: Draft 2019-09 use items, so we alse need support items!

**Comment by liangskyli** [comment]
@drwpow :You can also look at this！

https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation
image
In Draft 4 - 2019-09, tuple validation was handled by an alternate form of the items keyword. When items was an array of schemas instead of a single schema, it behaved the way prefixItems behaves.

**Review by drwpow — APPROVED** [review]
Thank you for correcting me! I had only used `prefixItems` before for the tuple type, not items. Glad to see `items` supports it too. This all looks good to merge, then!

**Pull request merged**